main_test.go

```diff
@@ -539,7 +539,7 @@ func TestProxyCluster(t *testing.T) {
 				proxyConf = strings.Replace(proxyConf, "<<LCADDRESS>>", etcdConf.LCAddress, -1)
 				proxyConf = strings.Replace(proxyConf, "<<ACADDRESS>>", etcdConf.ACAddress, -1)
 				proxyConf = strings.Replace(proxyConf, "<<MADDRESS>>", etcdConf.MAddress, -1)
-				proxyConf = strings.Replace(proxyConf, "<<UNHEALTHYTTL>>", strconv.FormatInt(int64(etcdConf.unhealthyMemberTTL), 10), -1)
+				proxyConf = strings.Replace(proxyConf, "<<UNHEALTHYTTL>>", strconv.FormatFloat(etcdConf.UnhealthyMemberTTL.Seconds()*1000, 'f', 2, 64), -1)
```

Comment:
can't you just do .Milliseconds() instead of .Seconds()*1000?

Reply:
Also, strconv.Itoa() would be easier?

Reply:
Unfortunately, Milliseconds isn't a method on time.Duration... I have no idea why that's the case.  

The returned value of Seconds() is a float64 so I could cast it to an int and use Itoa, but its just another conversion.

Reply:
I guess I could just change everything to seconds?  This is all specific to the test anyway...